file_store/src/error.rs

```diff
@@ -24,6 +22,12 @@ pub enum Error {
     Config(#[from] config::ConfigError),
     #[error("mpsc channel error")]
     Channel,
+    #[error("unsupported datarate {0}")]
+    UnsupportedDataRate(String),
+    #[error("unsupported invalid_reason {0}")]
+    UnsupportedInvalidReason(String),
+    #[error("unsupported participant_side {0}")]
+    UnsupportedParticipantSide(String),
```

Comment:
May want to make this more explicit..

Reply:
[c688769](https://github.com/helium/oracles/pull/215/commits/c688769c8b664bac808b180d82c0e7adb3cdd09f)